.circleci/config.yml

```diff
@@ -231,3 +231,39 @@ jobs:
                 --ignore tests/integration_tests/test_catalyst.py
 
       - run: *tests-mn
+
+  tests-rdbstorage:
+    docker:
+      - image: circleci/python:3.7
+      - image: circleci/redis:5.0.7
```

Comment:
Is redis still used?

Reply:
Oops. I removed it in https://github.com/optuna/optuna/pull/1497/commits/1a55a9515b34866330572c4f9e25c44f05e84d9f.